requirements.txt

```diff
@@ -24,3 +24,10 @@ sacremoses>=0.0.41
 protobuf<=3.20.1
 modlamp>=4.0.0
 PyTDC>=0.3.6
+torch-geometric<=2.0.3
```

Comment:
Since we are implementing from scratch, in case this upper bounds are coming rom GFlowNets, would it make sense to make everything compatible with the newest versions?

Reply:
Yes. But we also have some issues with `torchdrug` and `pytorch-scatter`